shared/src/state-migrations/migrations.ts

```diff
@@ -42,6 +47,44 @@ export function migrateStateExport(
     }
     return stateExport;
 }
+export function migratePartialExport(
+    partialExportToMigrate: PartialExport
+): Mutable<PartialExport> {
+    // Encapsulate the partial export in a state export and migrate it
+    const mutablePartialExport = cloneDeepMutable(partialExportToMigrate);
+    const stateExport = cloneDeepMutable(
+        new StateExport({
+            ...cloneDeepMutable(ExerciseState.create('123456')),
+            mapImageTemplates: mutablePartialExport.mapImageTemplates ?? [],
+            patientCategories: mutablePartialExport.patientCategories ?? [],
+            vehicleTemplates: mutablePartialExport.vehicleTemplates ?? [],
+        })
+    );
+    stateExport.fileVersion = mutablePartialExport.fileVersion;
+    stateExport.dataVersion = mutablePartialExport.dataVersion;
+    const migratedStateExport = migrateStateExport(stateExport as StateExport);
+    const mapImageTemplates =
+        mutablePartialExport.mapImageTemplates !== undefined
```

Comment:
Why are you checking for `undefined` here (same for patients and vehicles)? In the construction of the state export from the partial export, all undefined/null values are replaced by empty arrays.

I haven't tested it yet, but I think think that this can also cause unexpected overwrites, so that importing an export that only contains patients in overwrite-mode will delete all vehicles and map images from the current exercise.

Reply:
I'm checking to explicitly not have the circumstances described in your second paragraph. Mind you, I'm checking against the original partial export, not the state created from it. I will add a comment explaining this. This is explicitly to prevent overriding other categories with `undefined`.

Reply:
Yes, I also just noticed this. Thanks for the clarifying comment.